paasta_tools/cli/cmds/status.py

```diff
@@ -2643,10 +2655,9 @@ def paasta_status(args) -> int:
         tasks = [executor.submit(t[0], **t[1]) for t in tasks]  # type: ignore
         for future in concurrent.futures.as_completed(tasks):  # type: ignore
             return_code, output = future.result()
-            print("\n".join(output))
             return_codes.append(return_code)
 
-    return max(return_codes)
+    return max(return_codes), lock
```

Comment:
Are you returning the lock object here just for testing? I think this function needs to return just an int because it's called by https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/cli.py#L255 which is expecting an int.